ci-operator/config/stolostron/governance-policy-framework/stolostron-governance-policy-framework-main.yaml

```diff
@@ -33,7 +33,7 @@ tests:
     dependencies:
       COMPONENT_IMAGE_REF: grc-policy-framework-tests
     env:
-      ACM_RELEASE_VERSION: release-2.12
+      ACM_RELEASE_VERSION: release-2.13
```

Comment:
Set `PIPELINE_STAGE: integration` also. Without BVTs running, just setting the version isn't going to do it.